### ISSUE TITLE
"oc adm catalog mirror" set an invalid value for field meta.name for imagecontentsourcepolicy manifest

### ISSUE THREAD

**Issue by bmillemathias-1a** [openshift/oc#378]
When you create the mirror of the artifacts of the catalog, the content of the field meta.name of the manifest file is based on the path of the source registry, including / which are invalid.

so for the command

```
oc adm catalog mirror registry:5000/bmillemathias/olm/redhat-operators:v202004071506 \
 registry:5000/bmillemathias/olm/redhat-operators-build
```

You end with 
```
kind: ImageContentSourcePolicy
metadata:
  name: olm/redhat-operators
...
```

**Comment by jianzhangbjz** [comment]
@bmillemathias-1a Thank you for rasing this issue. One question, why not use the `registry:5000/bmillemathias/redhat-operators:v202004071506`?

**Comment by bmillemathias-1a** [comment]
Hello @jianzhangbjz 

The example I gave is a fictional, actually the path to the image is even longer, based on company organization.

**Comment by jianzhangbjz** [comment]
Ok, got it, thanks!

**Comment by shawn-hurley** [comment]
I believe that this is fixed as a side effect of #384. @bmillemathias-1a can you verify?

**Comment by bmillemathias-1a** [comment]
> 
> 
> I believe that this is fixed as a side effect of #384. @bmillemathias-1a can you verify?

sure I can, which is the next release the fix will be included in ?
Does the fix will hit 4.3 as well ?

**Comment by bmillemathias-1a** [comment]
> sure I can, which is the next release the fix will be included in ?

I meant I was looking for a compiled binary I can use to test.
